.github/workflows/unittest.yml

```diff
@@ -4,7 +4,7 @@ on: [ pull_request ]
 
 jobs:
   unit-tests:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
See https://github.com/apache/beam/issues/32766 – this is temporary. I'll switch it back when it's fixed.

Reply:
It's back to working – I'll remove the change